File: 2.Bolum/tekciftsayibulma/go/main.go
package main

import "fmt"

func main() {

	var sayi, k int
	fmt.Println("Lütfen bir sayı giriniz  ")
	fmt.Scanf("%d", &sayi)
	k = sayi % 2
	if k == 0 {
		fmt.Printf("Çift sayı")
	} else {
		fmt.Printf("Tek sayı")
	}

}

File: 2.Bolum/tekciftsayibulma/c/main.c
#include <stdio.h>
#include <stdlib.h>
void main()
{
    int sayi, k;
    printf("Bir sayi giriniz : ");
    scanf("%d", &sayi);
    k = sayi % 2;
    if (k == 0)
        printf("\n Cift sayi");
    else
        printf("\n Tek sayi");
    puts("\n Bitti");
}